plugin/cscope_maps.vim

```diff
@@ -6,6 +6,10 @@
 " plus some keyboard mappings that I've found useful.
 "
 " USAGE:
+" -- vim 8:     Stick this file in ~/.vim/pack/*/start/cscope/plugin/
```

Comment:
Thinking of it, this whole output *already* shown in ":help cscope", right? So there is no reason to show the installation details there. It seems to me it belongs to README.md file. If you agree, can you please extract all installation instructions from here to README.md, while at it?

Also, I think it's better to mention the command for generation of helptags, otherwise user won't be able to use ':help cscope'. It can be borrowed from here: https://github.com/vim-airline/vim-airline/blob/master/README.md#Installation

And also, I guess you can replace '*' with 'dist', the same as it's recommended in the vim-airline plugin (link above), just not to confuse the user with pain of choice :)